app/models/job.rb

```diff
@@ -14,7 +14,14 @@ def request_json
     job_type.as_json.merge(id: id)
   end
 
+  private
+
   def default_values
     self.status ||= "UNDEFINED"
   end
+
+  def send_email
```

Comment:
We probably only want to send an email if there's an event receiver listening for an error code and when there's an attached action for sending an email. Maybe next PR could start setting up a framework for that? no action required here though, just thinking ahead